maps/MODULES/binding.lua

```diff
@@ -46,18 +66,39 @@ town_mt = {
 			return GetTownOwner(t)
 		elseif k == "faction" then
 			return GetTownFaction(t)
+		elseif k == "x" then
+			return GetTownX(t)
+		elseif k == "y" then
+			return GetTownY(t)
+		elseif k == "visitingHeroIdx" then
```

Comment:
I'm guessing this is one of the ones you did while sleep deprived. One thing that happens on all-nighters is it's easy to just sit down and do things without without asking why or whether you need to.

Remove this one. As I've said, indices like this should not be exposed to the scripter.